bin/g

```diff
@@ -579,7 +585,24 @@ activate() {
 # Install <version>.
 #
 
-install() {
+install_version() {
+  version=$1
+  dir="$GO_VERSIONS_DIR/$version"
+
+  if [ -d "$dir" ]; then
+    set_version "$version"
+    exit
+  fi
+
+  download_version $version
+  set_version $version
```

Comment:
@feualpha let's add a log_info here with `installed` and `$(go version)`; this is mainly to confirm to the user the intended version was installed, in case something is off and the selected version is not actually activated.

Reply:
I mean this bit:

```

  if [ "$version" = "latest" ]; then
    version=$(get_latest_version)
  fi
```

It is being repeated quite a bit, but it is OK.